airflow/models/dag.py

```diff
@@ -2797,6 +2831,29 @@ def __repr__(self):
         return self.name
 
 
+class DagOwnerAttributes(Base):
+    """
+    Table defining different owner attributes. For example, a link for an owner that will be passed as
+    a hyperlink to the DAGs view
+    """
+
+    __tablename__ = "dag_owner_attributes"
+    dag_id = Column(
+        StringID(),
+        ForeignKey('dag.dag_id', name='dag.dag_id', ondelete='CASCADE'),
+        nullable=False,
+        primary_key=True,
+    )
+    owner = Column(String(100), primary_key=True, nullable=False)
```

Comment:
We need validation and documentation somewhere to communicate this new 100-character restriction. (Likely not going to affect anyone, but still)

Reply:
I changed it to 500 characters, just as the current Log table defines the owner :) Do you think we still need to add documentation and validation? And if so, does it need to be in this PR, or is it a separate enhancement? 
https://github.com/apache/airflow/blob/main/airflow/models/log.py#L38